Source/IGListCollectionViewLayout.mm

```diff
@@ -14,6 +14,41 @@
 
 #import <IGListKit/IGListAssert.h>
 
+static CGFloat UIEdgeInsetsLeadingInsetInDirection(UIEdgeInsets insets, UICollectionViewScrollDirection direction) {
+    return direction == UICollectionViewScrollDirectionVertical ? insets.top : insets.left;
+}
+
+static CGFloat UIEdgeInsetsTrailingInsetInDirection(UIEdgeInsets insets, UICollectionViewScrollDirection direction) {
+    return direction == UICollectionViewScrollDirectionVertical ? insets.bottom : insets.right;
+}
+
+static CGFloat CGPointGetCoordinateInDirection(CGPoint point, UICollectionViewScrollDirection direction) {
+    return direction == UICollectionViewScrollDirectionVertical ? point.y : point.x;
```

Comment:
Thoughts on using a `switch(...)` instead so the compiler will error if something ever changes? It likely wont, just good for longterm maintenance.